docs/src/Groups/groups.md

```diff
@@ -22,6 +22,138 @@ Julia supports the following types of groups:
 * `DirectProductOfGroups` = direct product of two groups
 * `AutomorphismGroup` = group of automorphisms over a group
 
-## Permutations groups
 
-In Julia, permutations are displayed as products of disjoint cycles, as in GAP.
+If `x` is an element of the group `G` of type `T`, then the type of `x` is `GAPGroupElement{T}`.
+
+## Basics
+
+### Elements of groups
+
+Given a group `G`, it is always possible to have access to some particular elements.
+
+* `one(G)` = the identity element of `G`.
+```@repl oscar
+one(symmetric_group(4))
+one(GL(2,3))
+```
+
+* `rand(G)`: returns a random element of the group `G`.
+
+For very large groups, the algorithm to ensure a uniform probability could be expensive, so there exists also the function `rand_pseudo`: this is much faster than `rand`, but the elements of `G` are not necessarily equally distributed.
+
+* `gens(G)`: returns a list of generators for the group `G`. To have access to the ``n``-th generator of the list, it is sufficient to type `G[n]` instead of `gens(G)[n]`.
+```@repl oscar
+G = symmetric_group(4);
+gens(G)
+G[1]
+```
+If the group `G` has been created as the subgroup of another group generated by a list of elements `L`, then the generating set returned by the function `gens` corresponds to `L`.
+
+!!! note
+    The output of `gens(G)` is not, in general, the minimal list of generators for `G`.
+
+### Operations
+
+Oscar supports the following operations and functions on group elements.
+
+* `*` = multiplication between two elements in a group.
+* `inv(x)` = `x^-1` the inverse of `x`.
+* `x/y` = the element `x*y^-1`.
+* `x^n` = the ``n``-th power of `x`. If ``n = 0``, the identity of the group is returned; if ``n < 0 ``, the ``n``-th power of the inverse of `x` is returned.
+* `isone(x)`: returns whether `x` is the identity of the group.
+* `conj(x,y)` = `x^y` = the conjugate of `x` by `y`, i.e. the element `x^-1*y*x`.
+* `comm(x,y)` = the commutator of `x` and `y`, i.e. the element `x^-1*y^-1*x*y`.
+
+
+## Subgroups
+
+The subgroup of a group `G` generated by the elements `x,y,...` are defined by the following instruction:
+
+* `sub(G, [x,y,...])` ;
+* `sub(x,y,...)`.
+
+This function returns two objects: a group `H`, that is the subgroup of `G` generated by the elements `x,y,...`, and the embedding homomorphism of `H` into `G`. The object `H` has the same type of `G`, and it has no memory of the "parent" group `G`: it is an independent group.
+```@repl oscar
+G = symmetric_group(4);
+H = sub(G,[cperm([1,2,3]),cperm([2,3,4])]);
+H[1] == alternating_group(4)
+```
+
+
+## Permutation groups
+
+Permutation groups can be defined as symmetric groups, alternating groups or their subgroups.
+```@repl oscar
+degree(symmetric_group(4))
+```
```

Comment:
I am not very fond of the `@repl` feature:
- it requires Julia to process all `@repl` examples just to be able to render the documentation, which means building the manual takes longer
- it misses an opportunity for further testing, as we never notice if Julia's output changes

Instead, please use `@jldoctest` (specifically, the REPL variant) as I suggested before; read https://juliadocs.github.io/Documenter.jl/stable/man/doctests/ for details